Reply/app/src/main/res/drawable/ic_search_bar_circle.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  Copyright (c) 2019 Google Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+  in compliance with the License. You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software distributed under the License
+  is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+  or implied. See the License for the specific language governing permissions and limitations under
+  the License.
+  -->
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+
+  <solid android:color="#FFFFFF" />
+
+  <padding
+      android:bottom="5dp"
+      android:left="5dp"
+      android:right="5dp"
+      android:top="5dp" />
+
+  <corners android:radius="30dp" />
```

Comment:
If you're looking to fully round the corners of this drawable, you can do `view height / 2` to get the "fully round" radius size. Since we know the search bar will be `56dp` tall I believe this should be `28dp`